commit a8efaee5e94cba668c4ee0a730d0f9e333f0a205
Author: Twinkle Goyal <quic_twingoya@quicinc.com>
Date:   Thu Jul 11 17:32:20 2024 +0530

    libsystemdq: Update debug option to debug-extra

    Update libsystemdq meson files as per new meson verison.

    Signed-off-by: Twinkle Goyal <quic_twingoya@quicinc.com>

diff --git a/meson.build b/meson.build
index 04331dd41a..4946231d02 100644
--- a/meson.build
+++ b/meson.build
@@ -764,7 +764,7 @@ substs.set('DEBUGTTY', get_option('debug-tty'))
 
 enable_debug_hashmap = false
 enable_debug_mmap_cache = false
-foreach name : get_option('debug')
+foreach name : get_option('debug-extra')
         if name == 'hashmap'
                 enable_debug_hashmap = true
         elif name == 'mmap-cache'
diff --git a/meson_options.txt b/meson_options.txt
index 16c1f2b2fa..dc7951b991 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -46,7 +46,7 @@ option('debug-shell', type : 'string', value : '/bin/sh',
        description : 'path to debug shell binary')
 option('debug-tty', type : 'string', value : '/dev/tty9',
        description : 'specify the tty device for debug shell')
-option('debug', type : 'array', choices : ['hashmap', 'mmap-cache'], value : [],
+option('debug-extra', type : 'array', choices : ['hashmap', 'mmap-cache'], value : [],
        description : 'enable extra debugging')
 option('memory-accounting-default', type : 'boolean',
        description : 'enable MemoryAccounting= by default')
